Rewards_FlowChart.drawio.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
we should call customer database to check the availability of the customer. We can decode the token and check whether the customerId sent in request and in token both are same. Change this logic and update the PR